packages/testcontainers/src/container-runtime/clients/container/container-client.ts

```diff
@@ -8,27 +8,46 @@ import Dockerode, {
 } from "dockerode";
 import { Readable } from "stream";
 import { ContainerStatus, ExecOptions, ExecResult } from "./types";
+import { ExecVerboseResult } from "../../../types";
 
 export interface ContainerClient {
   dockerode: Dockerode;
+
```

Comment:
Could you pls remove the newline changes to `container-client.ts` and `test-container.ts` to reduce the size of the PR?